### PR TITLE
Update PPP for v1.4: Jordan

### PR DESCRIPTION
**Pull request by jordankanghm**

### CODE REVIEW

File: docs/team/jordankanghm.md [hunk 1]
```diff
@@ -5,24 +5,68 @@ title: "Jordan Kang's Project Portfolio Page"
...
     * Developer Guide:
+      * Added use cases for event create feature. (https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/53/files)
+      * Added implementation details for event create feature, inclusive of UML class and sequence diagrams. (https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/115/files)
+      * , event add volunteer, event list volunteer and event remove volunteer features, inclusive of UML class, object and sequence diagrams. (https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/219/files)
 
-* **Community**: to be added soon
-
-* **Tools**: to be added soon
+* **Community**:
+  * Provided feedback on the PRs of team members (https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/67, https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/52, https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/67, https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/69, https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/82, https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/91)
```
Comment (brein62) [inline docs/team/jordankanghm.md:71]: duplicate PR (#67)

File: docs/team/jordankanghm.md [hunk 2]
```diff
@@ -5,24 +5,68 @@ title: "Jordan Kang's Project Portfolio Page"
...
+  * Our group also had a **deadline** to meet, which asserted some pressure on me as I was responsible for the **fundamental aspect of our application**, and I also had to ensure I had a Minimum Viable Product early so that my group members have time to implement their features as well. Although the result was expectedly buggy, I am grateful for the experience as it taught me to **work under pressure** and **think from different perspectives** when implementating new fields so that my group members can have an easier time implementing their features as well.
+
+* **New Feature**: Event Add Volunteer Feature
+  * Implemented the functionality of **assigning volunteers to events**. This is important as it helps volunteer coordinators **easily keep track** of which volunteers are participating in which event, easing the volunteer management process. The following shows the aspects involved when implementing this feature.
+    * **Creating a new field** for `Event`: `assignedVolunteers` which is a Set<Name> which stores the names of volunteers assigned to the event. I also created a `assignedEvents` field for `Volunteer` which works the same way.
```
Comment (brein62) [inline docs/team/jordankanghm.md:27]: `Set<Name>` should be in code form

File: docs/team/jordankanghm.md [hunk 3]
```diff
@@ -5,24 +5,68 @@ title: "Jordan Kang's Project Portfolio Page"
...
+    * **Parsing of parameter arguments**, such as ensuring that the `Event` and `Volunteer` indexes provided are valid.
+    * **Implementing the logic** for adding a volunteer to an event, such as checking if the volunteer is already assigned to the event and checking if any of the volunteer's assigned events clashes with the current event.
```
Comment (brein62) [inline docs/team/jordankanghm.md:29]: nice!

**Review by brein62 — APPROVED** [review]
LGTM! just some minor fixes and you should be good.

File: docs/team/jordankanghm.md (hunk 1 above)
Comment (jordankanghm) [reply docs/team/jordankanghm.md:71]: wah legend

File: docs/team/jordankanghm.md (hunk 2 above)
Comment (jordankanghm) [reply docs/team/jordankanghm.md:27]: aight

File: docs/team/jordankanghm.md (hunk 3 above)
Comment (jordankanghm) [reply docs/team/jordankanghm.md:29]: thanks!

File: docs/team/jordankanghm.md [hunk 4]
```diff
@@ -5,24 +5,68 @@ title: "Jordan Kang's Project Portfolio Page"
...
+    * **Creating new classes** to represent the fields of an event, namely `EventName`, `Role`, `DateTime` `Location`, `Material` and `Budget`.
+    * Implementing **checking for valid values** for the new classes created.
+    * Ensuring that the newly created `Event` can be **added to the Model seamlessly**.
+    * Ensuring that Event data can be **written into and retrieved from storage files** successfully.
+    * **Writing test cases** to ensure the feature works correctly.
 
-* **New Feature**: to be added soon
+  * As the original code we were provided with dealt with `Person` objects, I had to create and implement the logic for `Event` objects entirely as the main purpose of our app pertained to being able to **create volunteering events**. The experience was extremely **time-consuming** and took me a few days to implement as the existing code for `Person` was quite significant. However, it was also **informative** as it really allowed me to understand on a deeper level how the original code works and allowed me to implement the logic for `Event` more easily.
```
Comment (zacwong2151) [inline docs/team/jordankanghm.md:21]: `extremely **time-consuming** and took me a few days` indeed sir

Comment (jordankanghm) [reply docs/team/jordankanghm.md:21]: HAHAHAHAHA

File: docs/team/jordankanghm.md [hunk 5]
```diff
@@ -5,24 +5,73 @@ title: "Jordan Kang's Project Portfolio Page"
...
+    * **Parsing of parameter arguments**, such as ensuring that the event index provided is valid.
+    * **Implementing the logic** for listing the assigned volunteers, such as mapping the names in `assignedVolunteers` to the actual volunteer objects.
+    * **Integrating the resultant list with JavaFX** to display the volunteers in a single ListView panel.
+    * **Writing test cases** to ensure the feature works correctly.
+
+    * When implementing this feature, I deliberated whether I should display the volunteers in a panel next to the event display panel, or a pop-up window. In the end, I went with the former as although a pop-up window would be more **user-friendly**, it would affect the **integration with other commands**. Although the deliberating process was significantly time-consuming, it gave me more insights and aided my **forward-thinking capabilities**. Additionally, it also taught me to be **adaptable** when things don't go as planned and changes must be made on the fly.
```
Comment (ruo-x) [inline docs/team/jordankanghm.md:42]: wowie so lihai?

**Review by ruo-x — APPROVED** [review]
thanku jordan for ur efforts! gained some inspo for my own PPP as well :) HHAHAH

**Comment by jordankanghm** [comment]
> thanku jordan for ur efforts! gained some inspo for my own PPP as well :) HHAHAH

make sure credit me

**Pull request merged**